docs/reference/deploy/master-config-reference.rst

```diff
@@ -1163,6 +1177,54 @@ those partitions/queues.
    the HPC partition named ``defq_GPU`` with the ``gpu_type`` property set, and Slurm constraint
    associated with the feature ``XL675d`` used to identify the model type of the compute node.
 
+**********************************
+ ``additional_resource_managers``
+**********************************
+
+For Kubernetes, cluster administrators can define additional resource managers to connect the
+Determined master service to remote clusters. For a single resource manager or to define the default
+resource manager, do not define this under ``additional_resource_manager``; instead, use the primary
+``resource_manager`` key.
+
+Resource manager names must be unique among all resource managers.
+
+Additional resource managers must have defined resource pools under them. Resource pool names must
+be defined and must be unique among all resource pools in all resource managers. Resource pools for
+each additional resource manager are not defined at the root level but rather their respective
+element in the resource manager list.
+
+For example, to define three resource managers (one default, two additional):
+
+.. code:: yaml
+
+   resource_manager: # the default resource manager
+   resource_pool: # resource pools for the resource manager defined above.
+      pool_name: "foo"
+
+   additional_resource_managers:
+
+      -  resource_manager:
+
+      type: kubernetes # required, this feature is only for Kubernetes.
+      name: "bar" # required
+      resource_pools:
+         pool_name: "abc"
+
+      -  resource_manager:
+
+      type: kubernetes # required, this feature is only for Kubernetes.
+      name: "baz" # required
+      resource_pools:
+         pool_name: "def"
+
+``resource_manager``
+====================
+
+Optional. Follows the existing resource manager configuration pattern, with ``resource_pools``
+nested. There can be 'n'-many resource managers defined under the ``additional_resource_manager``
+key, following this pattern. Additional resource managers defined here require a ``name`` and
+``resource_pools``.
```

Comment:
Optional. This section allows for the definition of 'n'-many (multiple) resource managers under the ``additional_resource_manager`` key. It follows the existing resource manager configuration pattern where ``resource_pools`` is nested. Any additional resource managers defined here require a ``name`` and ``resource_pools``.

Reply:
resource_pools is not nested in the 'default' configuration for RM, only in the resource managers defined under additional_resource_managers -- how should I work that?

Reply:
``resource_manager``
====================

Optional. Defines 'n'-many (multiple) resource managers under the
``additional_resource_manager key``, following the existing resource manager configuration pattern. Each additional resource manager requires a name and a nested ``resource_pools`` section.